src/components/Home/Contact/Contact.jsx

```diff
@@ -4,10 +4,14 @@ import editIcon from "../../../assets/images/pencil.png";
 import { Wrapper } from "./Contact.styled";
 
 const ContactList = ({ theme }) => {
+  const randomNumberGenerator = () => {
+    return Math.floor(Math.random() * 100000 + 1);
+  };
```

Comment:
create a folder in src called `utils` then put this util there in the file name called `generateRandomNumber.js`

Reply:
changed code as per suggestion